src/components/columns-list/columns-list.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, useEffect, useState } from "react";
+import { Button, Input, Modal } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+import Column from "../column/column";
+import { useAppDispatch, useAppSelector } from "../../app/hooks";
+import {
+  columnsCreate,
+  columnsGetAll,
+  selectColumns,
+} from "../../features/columns/columns-slice";
+import "./columns-list.styles.scss";
+
+type ColumnsListProps = {
+  board: string | undefined;
+};
+
+const ColumnsList: FC<ColumnsListProps> = ({ board }) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [newColumnTitle, setNewColumnTitle] = useState("");
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch({ type: "tasks/resetTasks" });
+    dispatch(columnsGetAll(board ?? ""));
+  }, []);
+
+  const columns = useAppSelector(selectColumns);
+
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleOk = () => {
+    setIsModalOpen(false);
+    dispatch(
+      columnsCreate({
+        boardId: board ?? "",
+        request: { title: newColumnTitle || "New Column", order: 1 },
+      })
+    );
```

Comment:
```suggestion
if (board) {
    dispatch(
      columnsCreate({
        boardId: board,
        request: { title: newColumnTitle || "New Column", order: 1 },
      })
    );
}
```
т.к `boardId`  required

Reply:
и `order` должен быть по количеству колонок (length) или нет?